### ISSUE TITLE
Is there a way to install system packages in the dockerize step?

### ISSUE THREAD

**Issue by brspurri** [kyocum/disdat#197]
I need to have my workflow reference a system command that is installed via `apt-get install ...`.
Is there a way to allow the dockerize step to build a custom docker image?

Or another option that would work in my case would be to have the base image be something else. Not sure if this is possible either.

**Comment by kyocum** [comment]
Yes!  First, sorry, I'm so late getting to this.   Please feel free to ping me at kyocum@gmail.com if I don't get back sooner. 
You can list files in a standard deb.txt file.   You create a "config" directory, and place the deb.txt file within it.   Then run the dockerizer via `dsdt dockerize --config-dir <your dir>`.   There is also support for installing r packages.

**Comment by kyocum** [comment]
**Triage 2026-09-14: closing — answered, and the feature now lives in disdat-luigi.**

Answered above: place a `deb.txt` in a config directory and run `dsdt dockerize --config-dir <your dir>` (R packages are supported the same way).

Additionally, the dockerizer is no longer part of disdat core. This repo is now the core library only — it has no `dockerize` command, no `apply`, and no `PipeTask`. Container builds and pipeline execution moved to https://github.com/kyocum/disdat-luigi. Any follow-up on dockerizer behavior belongs there.
